11.hello-dapr-slim/README.md

```diff
@@ -0,0 +1,148 @@
+# Hello World(without docker)
+
+This tutorial will demonstrate how to get Dapr running locally on your machine without docker. We'll be deploying a Node.js app that subscribes to order messages demostrating service invocation capability of Dapr. The following architecture diagram illustrates the components that make up the first part sample: 
+
+![Architecture Diagram](./img/Architecture_DiagramA.png)
+
+## Prerequisites
+This sample requires you to have the following installed on your machine:
+- [Node.js version 8 or greater](https://nodejs.org/en/) 
+- [Postman](https://www.getpostman.com/) [Optional]
+
+## Step 1 - Setup Dapr 
+
+Follow [instructions](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#environment-setup) to download and install the Dapr CLI and initialize Dapr.
```

Comment:
Docker is mentioned as a prerequisite in the instructions linked here, can we provide instructions to use the --slim flag?

Reply:
modified in the docs repo PR